### PR TITLE
Add PyCon

### LINKED ISSUE #9: Add PyCon to the conference data tracker
Add PyCon with up-to-date information to the tracking spreadsheet in this repository.

Note: If for any reason an organizer refuses to answer questions, then we should add them to the tracking spreadsheet assuming they are not willing to sign the Speaker Engagement Agreement. They can always correct the information later, if it proves inaccurate.

### PR DESCRIPTION
**Pull request by ndren**
I got this data by contacting PyCon organizers.
Note: is partial the correct word here?
Closes #9.
/claim #9

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ The conference dataset hosted in this repository tracks the following informatio
 | [LambdaConf](https://lambdaconf.us) | Ziverge | Yes | Yes | Partial | Never |
 | [ZIO World](https://zioworld.com) | Ziverge | Yes | Yes | No | Never |
 | [ScalarConf](https://www.scalar-conf.com) | Software Mill | No | Yes | No | Never | 
+| [PyCon](https://pycon.org) | PyCon Board Committee | No | No | Yes | Partial |
```
Comment (jdegoes) [inline README.md:31]: It seems highly unlikely that PyCon would reimburse expenses, but not provide speakers with a free ticket. Can you confirm this?

Comment (ndren) [reply README.md:31]: Yes, I can confirm this. All speakers have to buy their own tickets, some (e.g. Keynote speakers) are compensated.

**Comment by ndren** [comment]
I added a specific category for that section, I think that should be a bit better.

File: README.md (hunk 1 above)
Comment (jdegoes) [reply README.md]: Finally, did they say they would not sign SEA if explicitly requested to do so by an invited speaker, or is that just the assumption?

Comment (ndren) [reply README.md]: They've said they would have to investigate when requested as they have never signed such a document before.

Comment (jdegoes) [reply README.md]: Perhaps we should put `Pending` in the column until they initiate and finish investigation?

Comment (ndren) [reply README.md]: That makes sense, I added that.

**Review by jdegoes — APPROVED** [review]

**Pull request merged**